src/main/java/duke/DukeTest.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+import org.junit.jupiter.api.Test;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class DukeTest {
+    @Test
+    public void ExitTest(){
```

Comment:
```suggestion
    public void exitTest(){
```
I believe test methods should still be in camelCase?